### ISSUE TITLE
Problems with quantile plots

### ISSUE THREAD

**Issue by Takashi-M891** [choishingwan/PRSice#370]
Hello.
Thank you very much for providing the PRSice-2 software. I have a question about its use.
I created a PRS for a continuous variable. According to the summary file, the best results were obtained when the threshold was 0.26, with a PRS R² of 0.17 and a P-value of 0.0075. However, in the quantile plot figure, I noticed that as the "Strata of the Polygenic Score" increase, there is a tendency for the "Change in phenotype given score in strata" to shift negatively.
I had expected a positive correlation, but is it possible for such a result to occur? I would appreciate it if you could provide some guidance on this matter.
Thank you.

Takashi

**Comment by choishingwan** [comment]
Yup, that is possible. R2 is bidirectional, and your p-value isn't really
significant. If you look at the coefficient, it should be negative. This
usually happen when you are doing cross trait analyses, or that your
overall analyses is not significant. Another thing you might want to check
is to see if you have specified the effective allele correctly.


Sam

**Comment by Takashi-M891** [comment]
To Sam

Thank you very much for your response. I see. Considering the aspect of multiple testing, this p-value is not significant, and such a result is possible. The effect allele was correctly specified. I sincerely appreciate your guidance.

Best regards,
Takashi
